res/values-ja/strings.xml

```diff
@@ -1063,20 +1063,20 @@
     <string name="CIRCLE">丸</string>
     <string name="SQUARES">SQUARES</string>
     <string name="TRICK_MODE">トリックモード</string>
-    <string name="ICE_SPLIT">ICE SPLIT</string>
+    <string name="ICE_SPLIT">アイススプリット</string>
```

Comment:
If you aren't about to translate others (Trick Split, Push Split, Pop Split etc), please keep here as it was, otherwise translate them all